server/search-api-helpers.ts

```diff
@@ -1,17 +1,18 @@
-const SITE_INDEX = process.env.ALGOLIA_SITE_INDEX_NAME;
-const BLOG_INDEX = process.env.ALGOLIA_BLOG_INDEX_NAME;
-const RESOURCES_INDEX = process.env.ALGOLIA_RESOURCES_INDEX_NAME;
-
-export type indexName = "blog" | "site" | "resources";
+const DOCS_INDEX = process.env.ALGOLIA_DOCS_INDEX_NAME;
 
 // Algolia API Parametres list: https://www.algolia.com/doc/api-reference/api-parameters/
-export const getSearchQuery = (indexName: indexName, query: string) => {
+export const getSearchQuery = (
```

Comment:
Q: Do we need separate `getSearchQuery` maybe just inline it in `getSearchQueries`?